llvm/unittests/FuzzMutate/StrategiesTest.cpp

```diff
@@ -640,4 +640,30 @@ TEST(ShuffleBlockStrategy, ShuffleLoop) {
     }";
   VerifyBlockShuffle(Source);
 }
+
+TEST(AllStrategies, SkipEHPad) {
+  StringRef Source = "\n\
+    define void @f(i32 %x) personality ptr @__CxxFrameHandler3 { \n\
+    entry: \n\
+      invoke void @g() to label %try.cont unwind label %catch.dispatch \n\
+    catch.dispatch: \n\
+      %0 = catchswitch within none [label %catch] unwind to caller \n\
+    catch: \n\
```

Comment:
Is connectToSink still replacing function calls with pointer calls?

Reply:
No, because `getCalledFunction` will return `nullptr` if it's a pointer.